docs/source/kernel-vanilla.md

```diff
@@ -0,0 +1,19 @@
+## Vanilla
```

Comment:
Although we use the term `vanilla` to describe a plain environment/kernel, I'm not sure that concept should be officially published by that name.  I'm thinking `Distributed` might be more appropriate, but I'd love to hear ideas.  

Btw, Yarn client mode is exactly this as well, we just happen to use spark-submit form different nodes in the cluster, that's all.